src/components/Resource/Forms/Training/TrainingImages.tsx

```diff
@@ -722,6 +763,23 @@ export const TrainingFormImages = ({ model }: { model: NonNullable<TrainingModel
           </Group>
         )}
 
+        {autoCaptioning.isRunning && (
+          <Paper
+            my="lg"
+            p="md"
+            withBorder
+            sx={{
+              backgroundColor:
+                theme.colorScheme === 'dark' ? theme.colors.dark[6] : theme.colors.gray[0],
+            }}
```

Comment:
If the change is just for a bgColor, let's use `style` instead. `sx` has been discontinued and it's in fact our main hassle to migrate (aside from `useStyles`).. Using `styles` is always ok :)